### ISSUE TITLE
Disable prettier

### ISSUE THREAD

**Issue by ghost** [stackblitz/core#2494]
### **Is your feature request related to a problem? Please describe.**
By default the prettier messes my code whenever I save it. Prettier is opinionated, opinionated things should no be enabled by default

### **Describe the solution you'd like**
At least an option to disable prettier completely for the project.

**Comment by pickle69420** [comment]
Yes this would be a great feature because prettier is pretty bad (in my opinion and from my experience)

**Comment by NG235** [comment]
Could you provide an example of Prettier changing your code in a way that isn't correct?

Additionally, it is also possible to disable Prettier formatting files on save by configuring workspace settings with the following:
`"editor.formatOnSave": false`

**Comment by pickle69420** [comment]
> Could you provide an example of Prettier changing your code in a way that isn't correct?
> 
> Additionally, it is also possible to disable Prettier formatting files on save by configuring workspace settings with the following: `"editor.formatOnSave": false`

Nope, this does nothing even on reload of workspace.

**Comment by NG235** [comment]
Workspace settings should look something like this:
```json
{
  "editor.formatOnSave": false
}
```

Make sure that it is formatted as a correct JSON object.

**Comment by NoralK** [comment]
I too cannot disable the wrapping.  If I put into Workspace Settings this setting it should disable the wrapping:
```
"prettier.singleAttributePerLine": false,
```
but it does not.  Is this by design, if so, why?  For me, it makes using Stackblitz unusable.
@kc0tlh @fvsch Could you please look at this.
